app/src/main/java/com/framgia/wsm/screen/requestleave/RequestLeaveViewModel.java

```diff
@@ -292,5 +461,9 @@ public boolean onSelection(MaterialDialog materialDialog, View view,
         int FORGOT_CARD_ALL_DAY = 8;
         int FORGOT_CARD_IN = 9;
         int FORGOT_CARD_OUT = 10;
+        int IN_LATE_WOMAN_M = 11;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.